src/System.Console/tests/System.Console.Tests.csproj

```diff
@@ -27,6 +27,9 @@
     <Compile Include="ThreadSafety.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="TermInfo.cs" />
+  </ItemGroup>
```

Comment:
Not sure and I can't seem to find my old builds...I'll check in the version I'm pushing out now